working_with_fhir/working_with_fhir_parsing.bal

```diff
@@ -1,22 +1,17 @@
+import ballerina/io;
 import ballerinax/health.fhir.r4.parser as fhirParser;
 import ballerinax/health.fhir.r4 as fhir;
-import ballerina/io;
-
-public function main() returns error? {
 
-    // The following example is a simple serialized Patient resource to parse
-    string input = "{" +
-    "\"resourceType\" : \"Patient\"," +
-    "  \"name\" : [{" +
-    "    \"family\": \"Simpson\"" +
-    "  }]" +
-    "}";
+final string patientData = string `{"resourceType" : "Patient", "name" : [{"family" : "Simpson"}]}`;
 
-    // Parse it - you can pass the input string or a json as input and the 
-    // type of the resource you want to parse.
+public function main(string input = patientData) returns error? {
```

Comment:
As per the above comment, this should allow `json|string`.

Reply:
we better leave it with only json and updated the PR